recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
+        "enable_precompiled_headers": [True, False],
+        "enable_tracy_profiling": [True, False],
+        "fPIC": [True, False],
+        "no_font_interface_default": [True, False],
+        "no_thirdparty_containers": [True, False],
+        "shared": [True, False]
+    }
+    default_options = {
+        "build_lua_bindings": False,
+        "disable_rtti_and_exceptions": False,
+        "enable_precompiled_headers": True,
+        "enable_tracy_profiling": False,
+        "fPIC": True,
+        "no_font_interface_default": False,
+        "no_thirdparty_containers": False,
+        "shared": False
+    }
+    generators = "cmake_find_package"
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
```

Comment:
You can add conan-io/conan#8002 here to set the minimum compliers (CCI supports compilers which do not offer C++14)

Reply:
I looked into ~/.conan/settings.yml and saw support for a lot more compilers than just AppleClang, Clang, GCC and MSVC. While I wouldn't want to overly restrict what compilers are given a chance, I also saw that only those four I mentioned before have cppstd properties configured. Does this essentially mean that using anything else is out of the question?

As such, wouldn't a simple check_min_cppstd call suffice by itself? I image that it'd also throw some kind of error if the property didn't exist in the first place.

Reply:
This is a fairly complicated problem... And I've taken the time to write it up (the best I know) in #5146 and document it....

For the TL;DR,

You are fairly close in your understanding!

the `cppstd` is not in the consumers profile and the setting is not managed by CCI ➡️ leads to a lot of stuff not working as you'd expect

Reply:
Thanks for the detailed explanation! I'll make sure to include this in the recipe. I've also extended the list of minimum compiler versions with those that were present in my settings.yml and had information about them on cppreference's compiler support page (https://en.cppreference.com/w/cpp/compiler_support/14).